### PR TITLE
Updates to definition

### LINKED ISSUE #49: Revise Hackspace Definition
After [this forum thread](https://forum.hackspace.org.uk/t/is-a-hackspace-not-open-to-all-still-a-hackspace/739) I've started a [Google Doc here](https://docs.google.com/document/d/1YyoHdaCyn_zhXaetj70MWVo0WkZ_qKTiwN_cGf7GoY8/edit#) to suggest revisions to the hackspace definition.

### PR DESCRIPTION
**Pull request by russss**
I've taken the suggestions from the [google doc](https://docs.google.com/document/d/1YyoHdaCyn_zhXaetj70MWVo0WkZ_qKTiwN_cGf7GoY8/edit) and done a bit of general re-writing, but hopefully I've captured most of the comments made. Please drop a comment here if you have any other suggestions.

I think these requirements should be kept fairly minimal - the main purpose of them is to make sure that member spaces are actual non-profits so that the benefits of the HSF aren't abused.

I've taken some wording from the [International Cooperative Alliance Principles](https://www.ica.coop/en/cooperatives/cooperative-identity).

I would like to keep this definition under review as we start accepting spaces, so please don't consider it set in stone. 

Fixes #49

### CODE REVIEW

**Review by unknowndomain — CHANGES_REQUESTED** [review]
I think it should be workshop, not workspace per last nights conversation.

File: define.md [hunk 1]
```diff
@@ -1,29 +1,18 @@
...
+    * has a clear, non-discriminatory process for members of the public to participate and become members 
+    * does not require affiliation with another organization, such as a university or employer, to be a full member
 
+Applications to join will be decided by the Membership Committee, with appeals decided by the Board.
 
-## Foundation Requirements
-
-* The UK Hackspace Foundation considers an organisation to be eligible to join as a [Member Space](structure.md), if it:
-    * is member owned/operated,
-    * is an unincorporated or incorporated association,
```
Comment (idnorton) [inline define.md:11]: Did we ditch this in favour of bank account?

**Review by idnorton — APPROVED** [review]
Looks good to me, streamlined the definition and answered a number of questions along the way.

Gets a :+1: from me

File: define.md (hunk 1 above)
Comment (idnorton) [reply define.md:11]: Discussed in telegram and this was replaced with:

> * is a non-profit association

Along with the requirement for a bank account which automatically requires some degree of legal structure so I'm happy.

**Comment by idnorton** [comment]
Thanks for your work on this @russss

**Review by castaway — COMMENTED** [review]
Looking good and much more generalised - I think we should ensure some key words (eg "association") are linked back to the glossary and listed with the specific meaning.. Or better, we add a second page that contains help notes on each of the items, as to how they are checked/verified, to ensure the meaning is understood

**Pull request merged**